File: Transport/OtherModels/farm/WorkCounter.cs
using Kdn.Ext.Attributes;

namespace Transport.OtherModels.farm
{
    public class WorkCounter
    {

        public const string FieldsMapping = @"[
                'CounterId',
                'CounterName'
            ]";

        [IdProperty]
        public int CounterId { get; set; }
        public string CounterName { get; set; }
    }
}

File: Transport/JS/Transport/combo/WorkCounter.js
T.combo.WorkCounter = Ext.extend(Kdn.form.ComboBox, {
    initComponent: function() {
        Ext.applyIf(this, {
            displayField: 'CounterName',
            valueField: 'CounterId',
            store: Kdn.ModelFactory.getStore('WorkCounter')
        });
        T.combo.WorkCounter.superclass.initComponent.call(this);
    }
});

Ext.reg('combo.workcounter', T.combo.WorkCounter);